server/chat-plugins/rock-paper-scissors.tsx

```diff
@@ -2,14 +2,14 @@
  * Rock Paper Scissors plugin by Mia
  * @author mia-pi-git
  */
-import {Utils} from '../../lib';
+import preact from 'preact';
 
 const MAX_ROUNDS = 200;
 const TIMEOUT = 10 * 1000;
-const ICONS: {[k: string]: string} = {
-	Rock: `<i class="fa fa-hand-rock-o"></i>`,
-	Paper: '<i class="fa fa-hand-paper-o"></i>',
-	Scissors: '<i class="fa fa-hand-scissors-o"></i>',
+const ICONS: {[k: string]: preact.VNode} = {
+	Rock: <i className="fa fa-hand-rock-o"></i>,
+	Paper: <i className="fa fa-hand-paper-o"></i>,
+	Scissors: <i className="fa fa-hand-scissors-o"></i>,
 };
```

Comment:
```suggestion
	Rock: <i class="fa fa-hand-rock-o"></i>,
	Paper: <i class="fa fa-hand-paper-o"></i>,
	Scissors: <i class="fa fa-hand-scissors-o"></i>,
};
```

The nice thing about Preact is that you can just use `class` instead of `className`

Reply:
Huh, I tried using `className` and TSC yelled at me.

Reply:
If TSC is yelling at you, you should make sure it's set up to use Preact JSX and not React JSX.

Reply:
How does that work? TSC only accepts shades of `react`.